hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestWatchForCommit.java

```diff
@@ -343,61 +349,24 @@ public void testWatchForCommitForRetryfailure() throws Exception {
     cluster.shutdownHddsDatanode(pipeline.getNodes().get(1));
     // again write data with more than max buffer limit. This wi
     try {
-      // just watch for a lo index which in not updated in the commitInfo Map
-      xceiverClient.watchForCommit(index + 1, 20000);
+      // just watch for a log index which in not updated in the commitInfo Map
+      // as well as there is no logIndex generate in Ratis.
+      // The basic idea here is just to test if its throws an exception.
+      xceiverClient
+          .watchForCommit(index + new Random().nextInt(100) + 10, 20000);
```

Comment:
instead of a Random increment, why not increment by a fixed number everytime - say 100 or 110? This applies to all the other modified test cases as well.

Reply:
The idea here is to run the test each time with unique number so any possible hacks/errors get caught if any.